dwmapi.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build windows
+// +build windows
+
+package win
+
+import "golang.org/x/sys/windows"
+
+type DWMWINDOWATTRIBUTE uint32
+
+const (
+	DWMWA_NCRENDERING_ENABLED            DWMWINDOWATTRIBUTE = 1
+	DWMWA_NCRENDERING_POLICY             DWMWINDOWATTRIBUTE = 2
+	DWMWA_TRANSITIONS_FORCEDISABLED      DWMWINDOWATTRIBUTE = 3
+	DWMWA_ALLOW_NCPAINT                  DWMWINDOWATTRIBUTE = 4
+	DWMWA_CAPTION_BUTTON_BOUNDS          DWMWINDOWATTRIBUTE = 5
+	DWMWA_NONCLIENT_RTL_LAYOUT           DWMWINDOWATTRIBUTE = 6
+	DWMWA_FORCE_ICONIC_REPRESENTATION    DWMWINDOWATTRIBUTE = 7
+	DWMWA_FLIP3D_POLICY                  DWMWINDOWATTRIBUTE = 8
+	DWMWA_EXTENDED_FRAME_BOUNDS          DWMWINDOWATTRIBUTE = 9
+	DWMWA_HAS_ICONIC_BITMAP              DWMWINDOWATTRIBUTE = 10
+	DWMWA_DISALLOW_PEEK                  DWMWINDOWATTRIBUTE = 11
+	DWMWA_EXCLUDED_FROM_PEEK             DWMWINDOWATTRIBUTE = 12
+	DWMWA_CLOAK                          DWMWINDOWATTRIBUTE = 13
+	DWMWA_CLOAKED                        DWMWINDOWATTRIBUTE = 14
+	DWMWA_FREEZE_REPRESENTATION          DWMWINDOWATTRIBUTE = 15
+	DWMWA_PASSIVE_UPDATE_MODE            DWMWINDOWATTRIBUTE = 16
+	DWMWA_USE_HOSTBACKDROPBRUSH          DWMWINDOWATTRIBUTE = 17
+	DWMWA_USE_IMMERSIVE_DARK_MODE        DWMWINDOWATTRIBUTE = 20
+	DWMWA_WINDOW_CORNER_PREFERENCE       DWMWINDOWATTRIBUTE = 33
+	DWMWA_BORDER_COLOR                   DWMWINDOWATTRIBUTE = 34
+	DWMWA_CAPTION_COLOR                  DWMWINDOWATTRIBUTE = 35
+	DWMWA_TEXT_COLOR                     DWMWINDOWATTRIBUTE = 36
+	DWMWA_VISIBLE_FRAME_BORDER_THICKNESS DWMWINDOWATTRIBUTE = 37
+	DWMWA_SYSTEMBACKDROP_TYPE            DWMWINDOWATTRIBUTE = 38
+)
+
+const (
+	DWM_CLOAKED_APP       uint32 = 1
+	DWM_CLOAKED_SHELL     uint32 = 2
+	DWM_CLOAKED_INHERITED uint32 = 3
```

Comment:
```suggestion
	DWM_CLOAKED_INHERITED uint32 = 0x4
```
Can you please also use hex notation for the preceding constants?